server/routes/v1/upload.ts

```diff
@@ -161,7 +162,7 @@ export const postUpload = [
       version.model = model._id
       await version.save()
 
-      req.log.info({ modelId: model._id }, 'Created model document')
+      req.log.info({ model: _.pick(model, [ '_id', 'uuid', 'schemaRef' ]) }, 'Created model document')
```

Comment:
Should be in serializer / service, not here.

Reply:
Now a serializer